med-web-app-front/src/services/pipeline.service.js

```diff
@@ -12,6 +12,10 @@ class PipelineService {
     savePipeline(jsonConfiguration, description) {
         return axios.post(API_URL + 'save', {jsonConfiguration, description},{ headers: authHeader() });
     }
+
+    deletePipeline(pipelineId) {

```

Comment:
Codacy has a fix for the issue: [Replace `····deletePipeline(pipelineId)·{␍` with `··deletePipeline(pipelineId)·{`](https://app.codacy.com/gh/danielsheh02/Medical-Web-App/pullRequest?prid=9534848)
```suggestion
  deletePipeline(pipelineId) {
```